### ISSUE TITLE
Investigate the sudden increase of loss at ~80k steps

### ISSUE THREAD

**Issue by carriex** [carriex/scone#2]
With the current training schedule, the loss suddenly increases from 0.089 to 1.21 at around 80k steps (72 epochs). 

Several directions for investigation / potential workaround:

- Data issue 
- Reduce learning rate: https://pytorch.org/docs/stable/optim.html#torch.optim.lr_scheduler.ReduceLROnPlateau
- Gradient clipping
